test/hotspot/jtreg/vmTestbase/nsk/jdi/MethodExitRequest/addClassExclusionFilter/filter001.java

```diff
@@ -74,6 +74,7 @@ public static int run (String argv[], PrintStream out) {
 
     private String classExclName1 = "java";
     private String classExclName2 = "sun";
+    private String classExclName3 = "jdk";
```

Comment:
The fix is reasonable. However,
I think it would be better to use List<String> with List.of(...) and list.contains() to filter exclusions.

Reply:
If you think it is easier, then fine.